content/docs/ai-tools/ai-workflows.md

```diff
@@ -0,0 +1,278 @@
+---
+title: AI Workflows
+icon: /docs/icons/ai.svg
+editions: ["OSS", "EE", "Cloud"]
+version: "1.0.0"
+---
+
+Build AI workflows with your preferred LLM.
```

Comment:
the main intention for AI Workflows was to show RAG (the ChatCompletions tasks with all the embedding providers, RAG chat to get responses grounded in your data etc) and this is missing entirely. no blocker but if you don't manage, would be great to add after the release @aj-emerich 🙏 

FWIW I think perhaps you can add a new page in this AI tools section for RAG workflows?